packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -200,13 +203,33 @@ export default defineComponent({
         ? 3
         : 2
     })
+    const fileDroppedBreadcrumb = async (data) => {
+      const spaceRootRoutePath = router.resolve(
```

Comment:
Last finding I think... the `AppBar` component is not only used in the `GenericSpace` view but in all files app views. This only works at the moment because all situations where a drag&drop to a parent folder works can only come from the `GenericSpace` view. Could you please emit the event (currently in line 220) with the `data.path` and handle all the rest of your function here in the `GenericSpace` view? I don't want the generic space view specifics to leak into the `AppBar` component.